vllm/worker/hpu_model_runner.py

```diff
@@ -1414,10 +1631,33 @@ def trim_attn_metadata(self, metadata: AttentionMetadata) -> object:
         # input_hash(123) != input_hash(321)
         # input_hash("abc") != input_hash("cba")
         attention_metadata = subtuple(metadata, 'TrimmedAttentionMetadata', [
-            'attn_bias', 'seq_lens_tensor', 'context_lens_tensor',
-            'block_list', 'block_mapping', 'block_usage', 'slot_mapping',
-            'is_prompt', 'block_indices', 'block_offsets', 'block_scales',
-            'block_groups'
+            'attn_bias',
+            'seq_lens_tensor',
+            'context_lens_tensor',
+            'block_list',
+            'block_mapping',
+            'block_usage',
+            'slot_mapping',
+            'is_prompt',
+            'block_indices',
+            'block_offsets',
+            'block_scales',
+            'block_groups',
+            'num_prefill_tokens',
+            'num_decode_tokens',
+            'num_prefills',
+            'seq_lens',
```

Comment:
won't this get hashed based on sequence lengths values? if so, you'll be constantly recapturing hpugraphs

Reply:
OK, I'll work on this to prevent the hpugraphs recapturing.

Reply:
@kzawora-intel The recapturing issue is resolved, with modification of [here](https://github.com/HabanaAI/vllm-fork/pull/491/files#diff-e93f4e5c975d07796dc7e9b5bf9012301e80d43c60c9eccad5bd911e3107f3deR1006) and update of [here](https://github.com/HabanaAI/vllm-fork/pull/491/files#diff-8a4e7bc179404d9deb5464e3f7815baae14d0a88d9f4ab0b84757d05f469274eR102-R110).
I have a refactor that bring in the enc_dec_model_runner module and support warm-up buckets for encoder inputs. Now the new performance data is listed in the description above. Could you please make a review again? Thanks.